### ISSUE TITLE
Addon fails to install (blueprint error)

### ISSUE THREAD

**Issue by eelke** [binhums/ember-cli-mapbox#36]
We've tried this on two separate systems, but installing fails in both cases.

```
ember install ember-cli-mapbox
version: 2.3.0
Installed packages for tooling via npm.
installing ember-cli-mapbox
The `ember generate <entity-name>` command requires an entity name to be specified. For more details, use `ember help`.

```

The addon gets downloaded to node_modules correctly but the bower components are not downloaded. Downloading those manually afterwards seems to work.

**Comment by binhums** [comment]
Thanks for raising

**Comment by binhums** [comment]
@eelke Should be fixed with the release of [0.3.4](https://github.com/binhums/ember-cli-mapbox/releases/tag/v0.3.4), thanks for reporting.

@kylenathan thanks for submitting the fix!

If either of you have any feedback about the addon I'm on [twitter](https://twitter.com/binhums) and the emberjs community slack as @binhums.
